test/execution/sql_filter_manager_test.cpp

```diff
@@ -135,7 +135,12 @@ TEST_F(FilterManagerTest, MixedTaatVaatFilterTest) {
 }
 
 // NOLINTNEXTLINE
+#if __APPLE__
+TEST_F(FilterManagerTest, DISABLED_AdaptiveCheckTest) {
```

Comment:
We should open an issue for this so we don't lose track of it.

Reply:
It did pass one run. I need to fiddle with the knob a bit.